docs/api/sources.md

```diff
@@ -0,0 +1,41 @@
+# Flag Sources
+
+Django-Flags provides a means to provide custom flag sources using the [`FLAG_SOURCES` setting](../settings/#flag_sources). Flag sources are simply classes implement a `get_flags` method that returns a dictionary of flag name keys with a list of [`Condition`](#conditioncondition-value-sourcenone-objnone) objects.
```

Comment:
I think you meant "Flag sources are classes that implement a `get_flags` method, and return a dictionary of flag name keys with a list of condition objects?